tools/schema_tools/prep_opsim.py

```diff
@@ -101,67 +138,214 @@ def offsetHex():
             vertex_count += 1
     return offsets
 
-def add_dither(database, simname, overwrite=True):
-    ## Add the Krughoff-Jones dithering pattern to the opsim output
-    ## adds three columns (ditheredRA, ditheredDec, and vertex) and indexes
-    #connect to the database
+def offsetRandom(noffsets, inHex=True):
+    """ 
+    Returns translational random dither offsets in radians.
+    
+    Required input
+    --------------
+    * noffsets: int: number of offsets needed.
+    
+    Optional input
+    --------------
+    * inHex: bool: set to False if dont want to restrict the dithers to within
+                   the hexagon inscribing the FOV. Default: True
+                  
+    """
+    import numpy as np
+    from lsst.sims.maf.stackers import inHexagon
```

Comment:
If OpSim depends on MAF, it's going to depend on everything (because MAF depends on `sims_coordUtils`, which depends on `afw`, which means we are getting all of the complicated parts of the DM stack).  `inHexagon` does looks to me like a generic enough function that we could move it to `sims_utils` (our lightest-weight sims package, which I think we are already moving towards OpSim depending on).  @mareuter does this sound reasonable to you?